common/libs/globalFuncs.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+export function makeBlob(mime: string, str: string | Object) {
+  str = typeof str === 'object' ? JSON.stringify(str) : str;
+  if (str === null) return '';
+  const blob = new Blob([str], {
+    type: mime
+  });
+  return window.URL.createObjectURL(blob);
+}
```

Comment:
Do we need to carry over that awful globalFuncs.js file? Can we just make this `makeBlob.js`?